modules/custom/az_digital_asset_library/az_digital_asset_library.module

```diff
@@ -27,3 +27,33 @@ function az_digital_asset_library_form_submit(array $form, FormStateInterface $f
 function az_digital_asset_library_form_entity_browser_az_digital_asset_library_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   $form['#submit'][] = 'az_digital_asset_library_form_submit';
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Hide default provided media type by IntelligenceBank from media select dialog.
+ *
+ * See @link https://github.com/az-digital/az_quickstart/issues/1741
+ */
+function az_digital_asset_library_preprocess_links__media_library_menu(&$variables) {
+  if (
+    isset($variables['links']) &&
+    isset($variables['links']['media-library-menu-ib_dam_embed']) &&
+    isset($variables['links']['media-library-menu-ib_dam_embed']['link'])
```

Comment:
It looks like we don't need a test for each key: 

- See the last line before Example 2: https://www.php.net/manual/en/function.isset.php
- Or [this StackOverflow answer](https://stackoverflow.com/questions/832408/how-to-check-if-a-deep-array-value-is-present)